### ISSUE TITLE
Telepresence 0.67 on Arch Linux

### ISSUE THREAD

**Issue by dhensen** [datawire/telepresence#297]
### What were you trying to do?

I'm running Arch Linux, but there is "official" telepresence installer.
I installed [telepresence 0.65-7 via AUR](https://aur.archlinux.org/packages/telepresence)
[exact version for referene](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=telepresence&id=a1ce735b57a664b3b1964760631d3438f89879da)

I already have minikube v0.22.0 running. I can `kubectl get pods` and I see my deployed pods.
Finally I ran `telepresence` which failed with following traceback.

So I forked the [AUR PKGBUILD for telepresence](https://aur.archlinux.org/packages/telepresence) and upgraded it to 0.67:
https://github.com/dhensen/telepresence-pkgbuild/commit/f2d6d4f6b72dc27404640e9a13fbd249d0de5cc3
(I'm a PKGBUILD noob)

With that PKGBUILD I built an installable package; installed it via arch's pkg-manager `pacman` and it installed successfully. 

Then same thing happened again.

### What did you expect to happen?

Have telepresence present me with a shell or a usage page.

### What happened instead?

See traceback for what happened. I'm (limited) available to help make this work on Arch Linux if you need extra hands.

### Automatically included information

Command line: `['/usr/bin/telepresence']`
Version: `0.67`
Python version: `3.6.2 (default, Jul 20 2017, 03:52:27) 
[GCC 7.1.1 20170630]`
kubectl version: `Client Version: v1.6.6`
oc version: `(error: [Errno 2] No such file or directory: 'oc')`
OS: `Linux liekaliek 4.12.12-1-ARCH #1 SMP PREEMPT Sun Sep 10 09:41:14 CEST 2017 x86_64 GNU/Linux`
Traceback:

```
Traceback (most recent call last):
  File "/usr/bin/telepresence", line 257, in call_f
    return f(*args, **kwargs)
  File "/usr/bin/telepresence", line 2363, in go
    runner, remote_info, args, env, subprocesses, socks_port, ssh
  File "/usr/bin/telepresence", line 2105, in run_local_command
    connect_sshuttle(runner, remote_info, args, subprocesses, env, ssh)
  File "/usr/bin/telepresence", line 1897, in connect_sshuttle
    get_hellotelepresence()
  File "/usr/bin/telepresence", line 1884, in get_hellotelepresence
    format(next(counter))
  File "/usr/bin/telepresence", line 395, in get_output
    result = unicode(check_output(*args, **kwargs).strip(), "utf-8")
  File "/usr/lib/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.6/subprocess.py", line 418, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['python3', '-c', "import socket; socket.gethostbyname('hellotelepresence135')"]' returned non-zero exit status 1.

```

Logs:

```
me or service not known
  26.5 TL | [156] Capturing: (['python3', '-c', "import socket; socket.gethostbyname('hellotelepresence132')"],)...
Traceback (most recent call last):
  File "<string>", line 1, in <module>
socket.gaierror: [Errno -2] Name or service not known
  26.7 TL | [157] Capturing: (['python3', '-c', "import socket; socket.gethostbyname('hellotelepresence133')"],)...
Traceback (most recent call last):
  File "<string>", line 1, in <module>
socket.gaierror: [Errno -2] Name or service not known
  26.8 TL | [158] Capturing: (['python3', '-c', "import socket; socket.gethostbyname('hellotelepresence134')"],)...
Traceback (most recent call last):
  File "<string>", line 1, in <module>
socket.gaierror: [Errno -2] Name or service not known
  27.0 TL | [159] Capturing: (['python3', '-c', "import socket; socket.gethostbyname('hellotelepresence135')"],)...
Traceback (most recent call last):
  File "<string>", line 1, in <module>
socket.gaierror: [Errno -2] Name or service not known

```

**Comment by dhensen** [comment]
I just went through the telepresence.log. I think this is the underlying problem:

```
sudo iptables -t nat -nL
[sudo] password for dino: 
iptables v1.6.1: can't initialize iptables table `nat': Table does not exist (do you need to insmod?)
Perhaps iptables or your kernel needs to be upgraded.
```

I'm going to figure out how to fix that and report back.

**Comment by ark3** [comment]
Thanks for the bug report and follow-up! Before you go very far down the rabbit hole with `iptables`, I have a quick request. Can you please try Telepresence with a remote cluster, i.e. not Minikube? If you see the same issue with a remote cluster, then we can eliminate one possible source of bad behavior (infinite looping of DNS lookups when using Minikube and method vpn-tcp under certain circumstances). 

If you don't have easy access to a remote cluster, [Kubernaut](https://github.com/datawire/kubernaut#kubernaut) can get you time-limited access to one quickly.

**Comment by dhensen** [comment]
Ok I figured out what the problem was. Earlier on today I did a system upgrade (arch = rolling release OS) without rebooting. That upgrade contained a kernel upgrade and that messed with some modules being available before rebooting. I tried rebooting, then starting minikube again and running `telepresence` I now have a shell and I can curl my http endpoint pods! 

@ark3 Thanks for pointing out Kubernaut though, did not know that one yet. Also I have been down in many rabbit holes lately, so happy this did not become another trip. :+1:
